shared_model/interfaces/transaction_responses/committed_tx_response.hpp

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IROHA_COMMITTED_TX_RESPONSE_HPP
+#define IROHA_COMMITTED_TX_RESPONSE_HPP
+
+#include "interfaces/transaction_responses/abstract_tx_response.hpp"
+
+namespace shared_model {
+  namespace interface {
+    /**
+     * Tx pipeline succeeded, tx is committed in ledger
+     */
+    class CommittedTxResponse : public AbstractTxResponse<CommittedTxResponse> {
```

Comment:
Is there any profits from responses being implemented via classes, not just `enum {response1, response2, ...}`?

Reply:
Variant and clean code for generic functions, such as `toString` and `makeOldModel`.